### DISCUSSION TITLE
How to render on Rstudio

### DISCUSSION THREAD

**Discussion by DrLynTaylor** [PSIAIMS/CAMIS#80] [General]
Hi All, Mia & I were discussing rendering QMD files in CAMIS - When we click render, it renders all Files and not just the one we are writing.  This is problematic because if there is an error in 1 file, then it fails before even getting to ours.  Can we render an individual file?  Anyone got any good practice suggestions ?

**Comment by ghost** [comment]
Hi @DrLynTaylor , I tried rendering the the anove.qmd file and I am able to render single file. Please point to which file you are trying to render so i can try rendering that file and check .

**Reply by DrLynTaylor** [reply]
This one was so long ago, maybe we consider it resolved now as I can't even remember which file I was having trouble with! . I think we can close the issue. Thanks so much for following up.
